### PR TITLE
[Benchmark] Simplify follower inner loop

### PR DESCRIPTION
**Pull request by SaveTheRbtz**
This addresses Patrick's comments from #2838.  As a side effect this diff also fixes the `txsTotalUnknown` counter.

### CODE REVIEW

File: integration/benchmark/follower.go [hunk 1]
```diff
@@ -114,38 +150,21 @@ Loop:
 		default:
 		}
 
-		GetBlockByHeightTime := time.Now()
+		getBlockByHeightTime := time.Now()
 		block, err := f.client.GetBlockByHeight(f.ctx, f.height+1)
```
Comment (pattyshack) [inline integration/benchmark/follower.go:154]: f.height read needs to be guarded

**Review by pattyshack — APPROVED** [review]
assuming you fix f.height thread safety

File: integration/benchmark/follower.go (hunk 1 above)
Comment (SaveTheRbtz) [reply integration/benchmark/follower.go:154]: Done in https://github.com/onflow/flow-go/pull/3074, thanks!

**Review by AndrewM-SDET — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Pull request merged**